### PR TITLE
Gimp

### PR DESCRIPTION
**Pull request by pyhalov**

### CODE REVIEW

**Comment by pyhalov** [comment]
This is a big "gimp update" PR. Gimp still requires python 2.7, so no luck here, but at least it can be 64-bit.
PR will be splited in several parts:
1) babl, libmypaint, mypaint-brushes
2) gegl
3) gimp
4) gtkam, sane-fronteds, xsane rebuilds.

**Review by AndWac — COMMENTED** [review]
You'll need to split this PR because you are adding new libraries which won't be available on the build system. Furthermore there are packages with new .so versions which might also need to be separated from the gimp build.

Sorry, I didn't see your comment before I wrote this. Please ignore it.

File: components/library/gegl/Makefile [hunk 1]
```diff
@@ -10,79 +10,73 @@
 #
```
Comment (AndWac) [inline components/library/gegl/Makefile:10]: gegl needs the updated babl:
meson.build:159:0: ERROR: Invalid version of dependency, need 'babl' ['>=0.1.78'] found '0.1.18'.

**Review by AndWac — APPROVED** [review]
I have installed gimp, gtkam, and xsane on my desktop.
Alas I don't have a supported scanner and no camera at hand. So I can only confirm the xsane and gtkam started but I wasn't able to further check them.
I opened some pictures (jpeg, tif, gif) and tried some color settings with them. All worked as expected.

**Comment by pyhalov** [comment]
Merged as several separate PRs: #5976 #5977 #5978 #5979

**Pull request closed without merging**